mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule ExAws.S3.DirectUpload.Mixfile do
   # Dependencies
   defp deps do
     [
-      {:poison, "~> 2.0 or ~> 3.0"},
+      {:poison, "~> 2.0 or ~> 3.0 or ~> 5.0"},
```

Comment:
I think that this can be simplified as `{:poison, ">= 2.0 and < 6.0"}`, although it may need to be `{:poison, ">= 2.0.0 and < 6.0.0"}`.